### PR TITLE
Fix(Sidebar and Content): Refactoring of side bar and fixing main con…

### PR DESCRIPTION
**Pull request by Eugene44-hub**
…tents on right side

### CODE REVIEW

**Review by rupali-codes — CHANGES_REQUESTED** [review]
Hey, just noticed a few changes for mobile or it to be responsive
1. sidebar is fixed, its not scrolling
2. it shud be closed when any element is clicked
3. content overlapping
image

File: components/SideNavbar/SideNavbar.tsx [hunk 1]
```diff
@@ -13,23 +13,18 @@ export const SideNavbar = () => {
...
-    <div className={`lg:w-1/5 bg-base-300 p-4 lg:min-h-screen`}>
+    <div className={`lg:w-[290px] bg-base-300 fixed top-0 left-0 w-full p-4 lg:h-full `}>
       <div className="flex justify-between">
-        <h1 className="text-3xl font-bold mb-4">
-          <span className="border-b-4 border-violet-500 text-violet-200">
-            Links
-          </span>
-          <span className="text-violet-500">Hub</span>
-        </h1>
+        <Logo className="text-3xl mb-4" />
```
Comment (Aadarsh805) [inline components/SideNavbar/SideNavbar.tsx:18]: Should make the Logo into a link to the home route

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Comment by rupali-codes** [comment]
Awesome,

**Comment by Eugene44-hub** [comment]
> Hey, just noticed a few changes for mobile or it to be responsive
> 
> 1. sidebar is fixed, its not scrolling
> 2. it shud be closed when any element is clicked
> 3. content overlapping
>    image

fixed

**Pull request merged**